src/java.base/share/classes/java/io/BufferedReader.java

```diff
@@ -397,14 +397,7 @@ public String readLine() throws IOException {
     }
 
     /**
-     * Skips characters.
-     *
-     * @param  n  The number of characters to skip
-     *
-     * @return    The number of characters actually skipped
-     *
-     * @throws     IllegalArgumentException  If {@code n} is negative.
-     * @throws     IOException  If an I/O error occurs
+     * {@inheritDoc}
```

Comment:
This appears to make the skip method disappear from the BufferedReader javadoc
and only show up as a method declared in Reader.

Reply:
This is intended and expected.